client/src/Classifier.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef classifier_h
+#define classifier_h
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/ml/ml.hpp>
+
+using namespace std;
+using namespace cv;
+
+class Classifier {
+  CvSVM svm;
+
+public:
+  Classifier(string filename) {
+    svm.load(filename.c_str());
+  }
+
+  // Guess the characters name from the image icon.
+  string classify(const Mat &image) {
```

Comment:
Just gonna say this once 😄 

I prefer to place the deference stuff along with the type (e.g. `const Mat& image` instead of `const Mat &image`). It just feels more right. I'm hoping you just didn't think you could do that and you definitely agree.